src/services/infra/DynamoDBService.ts

```diff
@@ -1,82 +1,45 @@
-import {
-  DeleteCommandOutput,
-  GetCommandOutput,
-  UpdateCommandOutput,
-} from "@aws-sdk/lib-dynamodb"
+import { DeleteCommandOutput } from "@aws-sdk/lib-dynamodb"
 import autoBind from "auto-bind"
 
 import { config } from "@config/config"
 
 import { SiteLaunchMessage } from "@root/../microservices/site-launch/shared/types"
 
-import DynamoDBClient, { UpdateParams } from "./DynamoDBClient"
+import DynamoDBClient from "./DynamoDBClient"
 
-const MOCK_LAUNCH: SiteLaunchMessage = {
-  repoName: "my-repo",
-  appId: "my-app",
-  primaryDomainSource: "example.com",
-  primaryDomainTarget: "myapp.example.com",
-  domainValidationSource: "example.com",
-  domainValidationTarget: "myapp.example.com",
-  requestorEmail: "john@example.com",
-  agencyEmail: "agency@example.com",
-  githubRedirectionUrl: "https://github.com/my-repo",
-  redirectionDomain: [
-    {
-      source: "example.com",
-      target: "myapp.example.com",
-      type: "A",
-    },
-  ],
-  status: { state: "pending", message: "PENDING_DURING_SITE_LAUNCH" },
-}
 export default class DynamoDBService {
   private readonly dynamoDBClient: DynamoDBClient
 
   private readonly TABLE_NAME: string
 
-  constructor() {
-    this.dynamoDBClient = new DynamoDBClient()
+  constructor(dynamoDBClient: DynamoDBClient) {
+    this.dynamoDBClient = dynamoDBClient
     this.TABLE_NAME = config.get("aws.dynamodb.siteLaunchTableName")
     autoBind(this)
   }
 
   async createItem(message: SiteLaunchMessage): Promise<void> {
-    await this.dynamoDBClient.createItem(this.TABLE_NAME, MOCK_LAUNCH)
+    await this.dynamoDBClient.createItem(this.TABLE_NAME, message)
   }
 
-  async getItem(message: SiteLaunchMessage): Promise<GetCommandOutput> {
-    return this.dynamoDBClient.getItem(this.TABLE_NAME, MOCK_LAUNCH.appId)
-  }
+  async getAllSuccessOrFailureLaunches(): Promise<SiteLaunchMessage[]> {
+    const entries = ((await this.dynamoDBClient.getAllItems(this.TABLE_NAME))
```

Comment:
<img width="651" alt="Screenshot 2023-05-18 at 8 27 55 AM" src="https://github.com/isomerpages/isomercms-backend/assets/42832651/d926aff0-7160-45d6-acf2-0e47bee0b7b1">
Somehow the typing that the SDK defines the object to have the [DynamoDB types](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.NamingRulesDataTypes.html), but the return value already has returned the object of our desired shape O.o 
Since we are the only ones adding the items to the table, the risk of the shape being different from expected is low. 
Open to feedback on this !

Reply:
how do we replicate this behaviour (or if difficult, could you just attach an ss of the `Item` itself so that we can see in entirety the properties)

Reply:
Have updated the PR desc to include the steps, but the TLDR when logging the result over is:

<img width="574" alt="Screenshot 2023-05-22 at 9 28 07 AM" src="https://github.com/isomerpages/isomercms-backend/assets/42832651/cb6ea799-946b-471c-b225-ac75214bab64">

For context, we should instead expect an object of a shape something similar to this:

```
{
  "appId": {
    "S": "my-app"
  },
  "agencyEmail": {
    "S": "agency@example.com"
  },
  "domainValidationSource": {
    "S": "example.com"
  },
  "domainValidationTarget": {
    "S": "myapp.example.com"
  },
  "githubRedirectionUrl": {
    "S": "https://github.com/my-repo"
  },
  "primaryDomainSource": {
    "S": "example.com"
  },
  "primaryDomainTarget": {
    "S": "myapp.example.com"
  },
  "PRIMARY_KEY": {
    "S": "my-app"
  },
  "redirectionDomain": {
    "L": [
      {
        "M": {
          "source": {
            "S": "example.com"
          },
          "target": {
            "S": "myapp.example.com"
          },
          "type": {
            "S": "A"
          }
        }
      }
    ]
  },
  "repoName": {
    "S": "my-repo"
  },
  "requestorEmail": {
    "S": "john@example.com"
  },
  "status": {
    "M": {
      "message": {
        "S": "PENDING_DURING_SITE_LAUNCH"
      },
      "state": {
        "S": "pending"
      }
    }
  }
}
```